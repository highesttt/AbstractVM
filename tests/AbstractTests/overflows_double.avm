push double(180000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000)
dump
exit
