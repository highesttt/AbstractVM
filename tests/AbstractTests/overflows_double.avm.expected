Return: 84
Error: [1;31mline 1:[1;33m Overflow or underflow on a value
